doc/usage.md

```diff
@@ -0,0 +1,75 @@
+# Usage
+
```

Comment:
```suggestion
Now that you have deployed Chaos Mesh in your environment. In this document, you will learn about how to use it for your chaos experiments.

```

Reply:
Remember what we talked about structure? You need to front load the most important information by adding the major points.